.github/workflows/update-nimbus-experiments.yml

```diff
@@ -0,0 +1,36 @@
+name: "Update Nimbus Experiments"
+
+on:
+  schedule:
+    - cron: '*/30 * * * *'
```

Comment:
Is there a reason you want to run this every 30min? 

Seems way too often for it to be scheduled for 30 min but if there is a good reason then I am all ears @jeddai

Reply:
It should be a no-op and exit quickly if there aren't any changes, but the point is the turn around time between the experiment launching and then landing in the repo to make it into the next build.  The major risk of this workflow is missing the timing between launching in Experimenter and the build being cut, so if we set this to something less aggressive like once a day we risk someone thinking they're in time to make it into the build and launching in experimenter and then the build process starting without the change.

We're just prototyping this workflow for now, so this is totally flexible in the future if we find we can get away with checking less frequently, but if it's doable to land at this frequency while we're still figuring out the workflow I think that's safer.